src/CommunityToolkit.Maui.MediaElement/Services/SafeFireAndForgetExtensions.shared.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace CommunityToolkit.Maui.Media.Services;
+
+/// <summary>
+/// Extension methods for System.Threading.Tasks.Task and System.Threading.Tasks.ValueTask
+/// </summary> 
+static partial class SafeFireAndForgetExtensions
```

Comment:
Can we use @brminnick  package? https://www.nuget.org/packages/AsyncAwaitBestPractices/

Reply:
Thanks for the shoutout! Let's avoid adding additional dependencies.

Just FYI - `CommunityToolkit.Maui.Extensions` also contains a copy/pasted implementation of `SafeFireAndForgetExtensions`.